packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -15,38 +15,38 @@
 
   display: inline-block;
   position: relative;
-  margin-block: var(--SpaceMarginBlock);
-  margin-inline: var(--SpaceMarginInline);
-  padding-block: var(--SpacePaddingBlock);
-  padding-inline: var(--SpacePaddingInline);
+  margin-block: var(--MarginBlock);
+  margin-inline: var(--MarginInline);
+  padding-block: var(--PaddingBlock);
+  padding-inline: var(--PaddingInline);
   transition-property: color, background-color, border-color, box-shadow;
   transition-duration: var(--TransitionDuration);
   transition-timing-function: var(--TransitionTiming);
   border-width: var(--BorderWidth);
   border-style: var(--BorderStyle);
-  border-color: var(--ColorBorderPrimary);
-  background-color: var(--ColorBackgroundPrimary);
-  color: var(--ColorText);
+  border-color: var(--PrimaryBorderColor);
+  background-color: var(--PrimaryBackgroundColor);
+  color: var(--TextColor);
   font-family: var(--FontFamily);
   font-size: var(--FontSize);
   font-weight: var(--FontWeight);
-  line-height: var(--FontLineHeight);
-  white-space: var(--FontWhiteSpace);
+  line-height: var(--LineHeight);
+  white-space: var(--WhiteSpace);
```

Comment:
we can remove this theme variable - it isn't part of what we want to expose see my note in slack

Reply:
How do we apply the appropriate token value here without a theme variable?

That is, even if this can't be themed, `LineHeight` needs to match `FontLineHeightBase`.

Reply:
sorry I was referring to `white-space` here. We talked this over with SIWNext team and the answer seemed to be nah, we dont need that

Reply:
That's good news - I was worried about ensuring correct behavior here.